acnportal/acnsim/interface.py

```diff
@@ -152,7 +152,7 @@ def is_feasible(self, load_currents, t=0, linear=False):
                 ignoring the phase angle and taking the absolute value of all load coefficients. Default False.
 
         Returns:
-            bool: If load_currents is feasible at time t according to this constraint set.
+            bool: If load_currents is feasible at time t according to this set of constraints.
         """
         return self._simulator.network.is_feasible(load_currents, t, linear)
```

Comment:
Do the conversion from a dict to a numpy array in this method. Also error check the schedule lengths here.